src/EditorFeatures/Core/Tagging/AsynchronousViewTaggerProvider.cs

```diff
@@ -21,16 +19,7 @@ protected AsynchronousViewTaggerProvider(IThreadingContext threadingContext, IAs
         {
         }
 
-        // TypeScript still is moving to calling the new constructor that does not take a IForegroundNotificationService.
-        // Until then, we can fetch one from another service of ours that already does. When TypeScript moves calling the
-        // new constructor, this should be deleted.
-        [Obsolete("This overload exists for TypeScript compatibility only and should not be used in new code.")]
-        protected AsynchronousViewTaggerProvider(IThreadingContext threadingContext, IAsynchronousOperationListener asyncListener, IForegroundNotificationService _)
-            : base(threadingContext, asyncListener)
-        {
-        }
```

Comment:
this constructor is no longer used by TS.  we moved them onto the constructor they should be using.